charts/kube-state-metrics/templates/role.yaml

```diff
@@ -11,14 +11,14 @@ metadata:
   labels:
     app.kubernetes.io/name: {{ template "kube-state-metrics.name" $ }}
     app.kubernetes.io/instance: {{ $.Release.Name }}
-    app.kubernetes.io/component: metrics 
+    app.kubernetes.io/component: metrics
     app.kubernetes.io/managed-by: {{ $.Release.Service }}
     app.kubernetes.io/version: "{{ $.Chart.Version }}"
-    app.kubernetes.io/part-of: {{ template "kube-state-metrics.name" $ }} 
-    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version }} 
+    app.kubernetes.io/part-of: {{ template "kube-state-metrics.name" $ }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version }}
     {{- if $.Values.customLabels }}
-    {{ toYaml $.Values.customLabels }}
-    {{- end }} 
+    {{ toYaml $.Values.customLabels | nindent 4 }}
```

Comment:
Just to make sure, did you have the chance to test it? :)

Reply:
i'm doing that and push again when it's done

Reply:
without customLabels :
````
Name:         stsdiscovery-dev01-208-monitoring-kube-state-metrics
Labels:       app.kubernetes.io/component=metrics
              app.kubernetes.io/instance=dev01-208-monitoring
              app.kubernetes.io/managed-by=Helm
              app.kubernetes.io/name=kube-state-metrics
              app.kubernetes.io/part-of=kube-state-metrics
              app.kubernetes.io/version=2.2.4
              argocd.argoproj.io/instance=dev01-208-monitoring
              helm.sh/chart=kube-state-metrics-4.1.0
Annotations:  <none>
PolicyRule:
  Resources          Non-Resource URLs  Resource Names                             Verbs
  ---------          -----------------  --------------                             -----
  statefulsets.apps  []                 [dev01-208-monitoring-kube-state-metrics]  [get list watch]
  pods               []                 []                                         [get]
````

with a custom Label :
````
Name:         stsdiscovery-dev01-208-monitoring-kube-state-metrics
Labels:       app.kubernetes.io/component=metrics
              app.kubernetes.io/instance=dev01-208-monitoring
              app.kubernetes.io/managed-by=Helm
              app.kubernetes.io/name=kube-state-metrics
              app.kubernetes.io/part-of=kube-state-metrics
              app.kubernetes.io/version=2.2.4
              argocd.argoproj.io/instance=dev01-208-monitoring
              helm.sh/chart=kube-state-metrics-4.1.1
              testCustomLabel=my-custom-label-for-test
Annotations:  <none>
PolicyRule:
  Resources          Non-Resource URLs  Resource Names                             Verbs
  ---------          -----------------  --------------                             -----
  statefulsets.apps  []                 [dev01-208-monitoring-kube-state-metrics]  [get list watch]
  pods               []                 []                                         [get]
````

Reply:
@mrueg so normally files are correctly formatted, tests are done and working, can you validate the PR